### ISSUE TITLE
[Suggestion] Prefab GUI

### ISSUE THREAD

**Issue by Xerma** [tsunamayo/Starship-EVO#4327]
Hi Tsuna! I thought of this while thinking of simple ways to place decoration in this hangar to give it some life and came up with this GUI concept as an upgrade for the prefab section in the Codex. Basically I think having categories and some detailed information about prefabs can go a very long way for players to really know what they're pasting around.

It would make most sense and be easiest for all players if there was a little + button to create a new category and have a separate smaller window pop-up and prompt the user to name the category. Categories could even have sub-categories for more flexibility(Drag a category into another category). Lastly it would also be helpful if players could drag the categories up and down to fit an order to their liking.

![PrefabGUISuggestion](https://user-images.githubusercontent.com/32974671/144759260-e4573f92-91e4-4507-9b4e-c4e423f1748d.png)

**Comment by thatisjake** [comment]
Also have categories for when spawning ships

**Comment by tsunamayo** [comment]
@Xerma Yes it makes sense, but I wanted to have a little preview on the right just like block with the same kind of info. Same would be for standard blueprints.
